src/js/views/right_panel_view.js

```diff
@@ -1162,8 +889,12 @@
         // TODO: Update each panel separately.
         update_img_css: function(zoom, dx, dy, save) {
 
-            dx = dx / (zoom/100);
-            dy = dy / (zoom/100);
+            var scaled_dx = dx / (zoom/100);
```

Comment:
could it be ``0``?

Reply:
No, the zoom is restricted to be 100% or more.